assignments/assignment-8/assignment-8.7.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment-8.7</title>
+    </head>
+    <body>
+        <h2>Photo contect application form</h2>
+
+        <label for="fullNameInput">Full Name</label>
+        <input
+            type="text"
+            placeholder="Full Name"
+            id="fullNameInput"
+            name="fullName"
+            required
+        />
+        <label for="emailInput">Email</label>
+        <input
+            type="email"
+            placeholder="Email"
+            id="emailInput"
+            name="email"
+        /><br /><br />
+
+        <label for="imageInput">Upload your photo</label>
+        <input
+            type="file"
+            required
+            id="imageInput"
+            name="image"
+            accept="image"
+        />
+        <label for="checkboxInput">I agree the terms of conditions</label>
+        <input type="checkbox" id="checkboxInput" name="agree" required />
```

Comment:
better choice for attr for,id="agreementCheckboxInput"
name="agreement"

Reply:
sure